docs/getting-started/signing-transactions.md

```diff
@@ -3,86 +3,227 @@ sidebar_position: 3
+```js
+const signed = result.filter(txn => txn != undefined).map(txn => btoa(String.fromCharCode.apply(null, txn)))
+```
+This results can then be sent to the `SubmitTransaction` endpoint for submission.
 
-Start your site on the French locale:
 
-```bash
-npm run start -- --locale fr
+## Signing with MyAlgoWallet
+Another way to sign transaction is by using MyAlgoWallet and MyAlgoConnect. Firstly, the myalgo script must be imported:
+```html
+<script type="text/javascript" src="/img/https://github.com/randlabs/myalgo-connect/releases/download/v1.1.1/myalgo.min.js"></script>
 ```
+### Connecting to the wallet and getting the addresses
+Before being able to connect to a wallet, an instance of MyAlgoConnect must be created.
+```js
+const wallet = new MyAlgoConnect();
+```
+Then, by using that instance, it is possible to run
+```js
+const accounts = (await wallet.connect()).map(account => account.address);
+```
+to connect to the wallet and get the addresses of all connected accounts.
 
-Your localized site is accessible at `http://localhost:3000/fr/` and the `Getting Started` page is translated.
-
-:::caution
+When run, the user will be prompted with a popup asking for their wallet's password
 
-In development, you can only use one locale at a same time.
+<img src="/img/myalgoconnect_connect.png" height="500px"/>
 
-:::
+After inserting their password, the prompt will ask the user to choose which accounts to connect
 
-## Add a Locale Dropdown
+<img src="/img/myalgoconnect_select.png" height="500px"/>
 
-To navigate seamlessly across languages, add a locale dropdown.
+### Signing the tranasctions
+To sign the transactions, the `signTransaction` method can be used.
+```js
+const result = await wallet.signTransaction(toSign)
+```
+When run, a popup wil appear, asking the user to confirm if they are fine with signing the shown transaction. Once accepted, the user is asked once more for their password.
 
-Modify the `docusaurus.config.js` file:
+<img src="/img/myalgoconnect_sign.png" height="500px"/>
 
-```js title="docusaurus.config.js"
-module.exports = {
-  themeConfig: {
-    navbar: {
-      items: [
-        // highlight-start
-        {
-          type: 'localeDropdown',
-        },
-        // highlight-end
-      ],
-    },
-  },
-};
+Once signed, it is possible to encode the signed transactions to base64
+```js
+const signed = result.map(txn => btoa(String.fromCharCode.apply(null, txn.blob)));
 ```
+so that they can be sent to the `SubmitTransaction` endpoint.
 
-The locale dropdown now appears in your navbar:
+## Signing with AlgoSigner
+The last wallet for which we'll show how to sign transactions is AlgoSigner. To work with AlgoSigner, first, the AlgoSigner script must be imported:
+```html
+<script type="text/javascript" src="/img/chrome-extension://kmmolakhbgdlpkjkcjkebenjheonagdm/AlgoSigner.min.js"></script>
```

Comment:
Is it a typo? Algosigner is already injected to pages and is available via `window.AlgoSigner`

Reply:
I think you're right, I'll double check tomorrow and change it